src/cljs/main/broadfcui/common/notifications.cljs

```diff
@@ -115,6 +115,158 @@
    (fn [{:keys [state]} alert]
      (swap! state update :service-alerts #(filter (partial not= alert) %)))})
 
+(react/defc TrialAlertContainer
```

Comment:
#1519 deleted the `TrialAlertContainer` component; I'm now putting it back. Might be easiest to compare this to the commit prior to  #1519: https://github.com/broadinstitute/firecloud-ui/compare/a4d39cc0f197a70b87c1b731891722ab73473ca2...da_trialBannersBack